### ISSUE TITLE
ctrl+shift+h still fails for toggling state of AskQuestion experience

### ISSUE THREAD

**Issue by accesswatch** [microsoft/vscode#295767]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 
- OS Version: 

Steps to Reproduce:

Hi,

The issue with ctrl+shift+h continues to persist in the latest VSCode release.

I have a suggestion to resolve the issue. Change the Search in Files hotkey and add this when condition:

!inChatInput

In this case, if the chat window is in focus then it does not fire that specific keystroke. This resolved the issue for me.

The other solution may be to tie agent constants as a part of the when condition, similar to how the ctrl+shift+t hotkey is done for todos.

Jeff

**Comment by accesswatch** [comment]
@meganrogge I wasn't sure if you got ping'd on closed issues or not so opened a new issue on the problem.
